app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RecipesController < ApplicationController
+  before_action :find_query
+
+  def index
+    @recipes = EdamamApiWrapper.search_recipes(find_query).paginate(page: params[:page], per_page: 10)
```

Comment:
You've written your API wrapper to raise an error if the API call fails, but you're not looking for an error here. You should wrap this call in a `begin`/`rescue`.

This causes your pages to break on bogus data, rather than redirect.